packages/terra-clinical-detail-view/tests/jest/__snapshots__/DetailList.test.jsx.snap

```diff
@@ -2,73 +2,85 @@
 
 exports[`should render a default component 1`] = `
 <div
-  data-terra-clincial-detail-list="true"
+  data-terra-clinical-detail-list="true"
 >
-  <div
+  <ul
     class="list"
   />
 </div>
 `;
 
 exports[`should render a list 1`] = `
 <div
-  data-terra-clincial-detail-list="true"
+  data-terra-clinical-detail-list="true"
 >
-  <div
+  <ul
     class="list"
   >
-    <div
-      class="detail-list-item"
+    <li
+      class="list-item"
     >
-      <p>
-        Test
-      </p>
-    </div>
-    <div
-      class="detail-list-item"
+      <div
+        class="detail-list-item"
+      >
+        <p>
+          Test
+        </p>
+      </div>
+    </li>
+    <li
+      class="list-item"
     >
-      <p>
-        Test
-      </p>
-    </div>
-  </div>
+      <div
+        class="detail-list-item"
+      >
+        <p>
+          Test
+        </p>
+      </div>
+    </li>
+  </ul>
 </div>
 `;
 
 exports[`should render a title 1`] = `
 <div
-  data-terra-clincial-detail-list="true"
+  data-terra-clinical-detail-list="true"
 >
   <div
     class="title"
   >
     Title
   </div>
-  <div
+  <ul
     class="list"
   />
 </div>
 `;
 
 exports[`should render a title and a list 1`] = `
 <div
-  data-terra-clincial-detail-list="true"
+  data-terra-clinical-detail-list="true"
 >
   <div
     class="title"
   >
     Title
   </div>
-  <div
+  <ul
     class="list"
   >
-    <div
-      class="detail-list-item"
+    <li
+      class="list-item"
     >
-      <p>
-        Test
-      </p>
-    </div>
-  </div>
+      <div
+        class="detail-list-item"
+      >
+        <p>
+          Test
+        </p>
+      </div>
+    </li>
+  </ul>
 </div>
 `;
```

Comment:
Please add new tests for when using description list with terra-clinical-label-value component.

Reply:
Updated here: 15cddd9279f1479e0dbd5c474c63d85bb20d855b